program-workflows/how-to-enter-recency-data.md

```diff
@@ -0,0 +1,23 @@
+##HOW TO ENTER RECENCY DATA WITHIN THE HTS CLIENT CARD
```

Comment:
@ODORA0 

1. Do not use capitals in the heading, plus move this to HTS Card Data Entry

2. Also add how to enter HTS card only (in the UgandaEMR Data Entry section)

Reply:
@ssmusoke Done

Reply:
@ODORA0 the title should be "How To Enter Recency Data from The HTS Client Card